modules/commons/commons-config/src/main/java/org/eclipse/dirigible/commons/config/Configuration.java

```diff
@@ -448,7 +448,8 @@ public static void setSystemProperty(String key, String value) {
 			"DIRIGIBLE_PRODUCT_COMMIT_ID",
 			"DIRIGIBLE_PRODUCT_TYPE",
 			"DIRIGIBLE_INSTANCE_NAME",
-			"DIRIGIBLE_SPARK_CLIENT_URI"
+			"DIRIGIBLE_SPARK_CLIENT_URI",
+			"DIRIGIBLE_EXEC_DISABLE_COMMAND_LOGGING"
```

Comment:
Let's align with the other environment variables:

`DIRIGIBLE_EXEC_DISABLE_COMMAND_LOGGING` -> `DIRIGIBLE_EXEC_COMMAND_LOGGING_ENABLED`